lib/tile_bag.rb

```diff
@@ -0,0 +1,40 @@
+module Scrabble
+  class TileBag
+    attr_reader :tile_bag
+
+    def initialize
+      @tile_bag = Scrabble::TileBag.create_tile_bag
+    end
+
+    def self.create_tile_bag
+      letters = ("a".."z").to_a
+      quantity= [9,2,2,4,12,2,3,2,9,1,1,4,2,6,8,2,1,6,4,6,4,2,2,1,2,1]
```

Comment:
I like the idea here, but this representation is hard to read / modify. What if you needed to change the number of "N"s - you'd have to count to figure out which one it was.

In this case, it might be better to be more verbose, something like

```ruby
tile_bag = []
tile_bag += ['A'] * 9
tile_bag += ['B'] * 2
# ... etc ...
```